### ISSUE TITLE
Error is thrown on projects without Pint installed

### ISSUE THREAD

**Issue by driesvints** [open-southeners/vscode-laravel-pint#1]
The following error is thrown on every save of every file for projects that don't have Pint installed. This is a bit annoying and counterintuitive. I feel that no warning should be thrown here.

<img width="493" alt="Screenshot 2022-06-23 at 16 27 35" src="https://user-images.githubusercontent.com/594614/175323944-e0f5ab36-5f44-43c7-868c-c518f56804ad.png">

**Comment by d8vjork** [comment]
Yeah, actually I should enhance the config of this even more to enable/disable auto-formatting on save and handle this situation.

I'll have this in mind, thanks for the report

**Comment by driesvints** [comment]
Thanks!

**Comment by aliwesome** [comment]
I'm facing this issue on Windows 10 too.
![image](https://user-images.githubusercontent.com/21131502/175765227-88d3d239-9726-4c2f-bd61-0b965e4cff1d.png)

**Comment by d8vjork** [comment]
Oh wow, I'll create another issue as it seems to be more related to the formatting command
